lib/src/version_js.dart

```diff
@@ -0,0 +1,3 @@
+/// Hardcoded dart version as `Platform` from `dart:io` is not available when
+/// targeting javascript.
+final dartVersion = '2.0.0';
```

Comment:
Is there really no way of getting version no in dart:web? cc @vsmenon

Reply:
We'd have to compile it into the output JS. Not currently done. We could include it with future plans WRT `dart:platform` etc – CC @lrhn

Reply:
Got it

Reply:
See https://github.com/dart-lang/sdk/issues/35969 - I'm hoping we can get consensus and land something in the coming milestone.